benchmarks/graphics_pipeline/main.cpp

```diff
@@ -379,8 +404,14 @@ void ProjApp::Setup()
         const uint32_t sphereVertexCount = mesh.GetCountPositions();
         const uint32_t sphereTriCount    = mesh.GetCountTriangles();
 
-        Geometry geo;
-        PPX_CHECKED_CALL(Geometry::Create(GeometryOptions::InterleavedU32().AddTexCoord().AddNormal().AddTangent(), &geo));
+        Geometry        lowPrecision;
+        GeometryOptions options;
+        options.isVertexDataCompressed = true;
+        options.IndexTypeU32().AddPosition(grfx::FORMAT_R16G16B16_FLOAT).AddTexCoord(grfx::FORMAT_R16G16_FLOAT).AddNormal(grfx::FORMAT_R8G8B8_SNORM).AddTangent(grfx::FORMAT_R8G8B8A8_SNORM);
```

Comment:
is there a reason why you use rgb8 for normal but rgba8 for tangent? normally rgba8 should be fine. better to be aligned to 4 bytes

Reply:
Updated.